README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
This seems like a confusing name for the described functionality. I'll need to read the blog post for more context so I can make a more useful comment.